push/templates/push/validation.html

```diff
@@ -11,19 +11,53 @@
 
 {% block content %}
     {% if not app.valid %}
-    <p>{% trans "To verify you own the signing key for your app, please sign the following token using the same key you use to sign this application's VAPID JWT, and paste the url-safe, base64-encoded value below." %}</p>
+    <p>{% trans "To verify you own the signing key for your app, please sign a JWT with the following claim using your app's VAPID signing key. Paste the encoded JWT below." %}</p>
         <dl>
-            <dt>{% trans "Token:" %}</dt>
+            <dt>{% trans "Payload:" %}</dt>
             <dd class="input-group">
               <span class="input-group-label"><a href="#" class="js-copy-button" data-clipboard-text="{{ app.vapid_key_token }}"><i class="fa fa-clipboard"></i></a></span>
               <input class="input-group-field" id="vapid-key-token" type="text" value="{{ app.vapid_key_token }}" readonly></input>
             </dd>
+
+            <dt>{% trans "Examples:" %}</dt>
+            <dd>
+        <ul class="tabs" data-tabs id="code-example-tabs">
+            <li class="tabs-title is-active"><a href="#python-jose" aria-selected="true">python-jose</a></li>
+            <li class="tabs-title"><a href="#node-jws">node-jws</a></li>
+        </ul>
+
+        <div class="tabs-content" data-tabs-content="code-example-tabs">
+            <div class="tabs-panel code-panel is-active" id="python-jose">
+                <pre><code class="python">from jose import jws
+
+private_key = &lt;your-private-key&gt;
```

Comment:
We may want to add a tad to this, just because this is really bare bones. 
maybe add
```
import ecdsa
private_key = ecdsa.SigningKey.from_pem(<i>"VAPID_key.pem"</i>)
```
so that they know where the private key comes from.